docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: '3'
+
+services:
+  hugo:
+    image: juristr/hugo-docker:latest
+    working_dir: /srv/hugo
+    volumes:
+      - .:/srv/hugo
+    command: "hugo server --bind 0.0.0.0 --templateMetrics"
+    ports:
+      - 1313:1313
```

Comment:
What does the docker-compose file do for us? If we can/should run docker in this repo for some reason, it would be good to have a (brief) readme.

Reply:
@noisecapella and I discussed this.  It allows us to assure that people trying this out locally get the latest version of hugo regardless of their host OS.  Ubuntu 18.04, for example, installs a very old version of hugo if you use their default repo.